dash_bio/package-info.json

```diff
@@ -18,14 +18,10 @@
     "lint": "eslint src --fix",
     "test": "npm run lint:test && npm run format:test && npm run test-unit",
     "test-unit": "jest",
-    "build:js-dev": "webpack --mode development",
     "build:js": "npm run lint && npm run format && webpack --mode production",
-    "build:py": "node ./extract-meta src/lib/components > dash_bio/metadata.json && python get_version_info.py && python -c \"import dash; dash.development.component_loader.generate_classes('dash_bio', 'dash_bio/metadata.json')\"",
-    "build:r": "dash-generate-components ./src/lib/components dash_bio -p package-info.json --r-prefix='dashbio' --r-imports='dash' --r-depends='manhattanly'",
-    "build": "npm run build:js && npm run build:py",
-    "build:all": "npm run build && npm run build:r",
-    "postbuild": "es-check es5 dash_bio/*.js",
-    "build:dev": "npm run build:js-dev && npm run build:py"
+    "build:backends": "node ./extract-meta src/lib/components > dash_bio/metadata.json && python get_version_info.py && dash-generate-components ./src/lib/components dash_bio -p package-info.json --r-prefix='dashbio' --r-imports='dash' --r-depends='manhattanly'",
```

Comment:
`extract-meta` is now part of [dash-generate-components](https://github.com/plotly/dash/blob/efc42b46fef033bd4ec1f082cfe5cbab97a62646/dash/development/component_generator.py#L58-L77) so I think we can remove that (and in fact remove that file from this repo 🎉 )

I believe the same is true for `get_version_info` - if you look at the `package-info.json` that's present in the `dash-bio` dir, that's the full `package.json` as copied by `dash-generate-components`, not the partial one from `get_version_info`.

Reply:
That makes a lot of sense - it's nice that `dash-generate-component` bundles a lot of these parts of the build process together; I wasn't entirely sure which of these commands were deprecated. 

Updated in 4e6cf1d.